components/AUPage/CTA.jsx

```diff
@@ -170,7 +177,7 @@ export const CTA = () => (
                     )}
                   </div>
                 </div>
-                <div className="max-sm:hidden">
+                <div className="hidden md:block">
```

Comment:
fixed this 
<img width="643" alt="image" src="https://github.com/user-attachments/assets/dd48c245-ea5d-4935-b997-8bfbce6d5f2e" />